config/software/bash.rb

```diff
@@ -22,6 +22,8 @@
 
 version("4.3.30") { source md5: "a27b3ee9be83bd3ba448c0ff52b28447" }
 
+license :project_license
```

Comment:
I think this should be:

```
license "GPL-3.0"
license_file "COPYING"
```

Found this out by downloading and inspecting https://ftp.gnu.org/gnu/bash/bash-4.3.30.tar.gz